styles/stepper.css

```diff
@@ -104,3 +104,4 @@
         padding-left: 1rem;
     }
 }
+
```

Comment:
How come this is a part of the PR? Don't think this needs to be here if there isn't any changes to this file

Reply:
file got reverted